data/team.yml

```diff
@@ -2950,3 +2950,14 @@ dora_grgic:
   location: Salzburg, Austria
   links: '[LinkedIn](https://www.linkedin.com/in/doragrgic/)'
   description: 'Dora is an experienced Informaiton Security and Compliance professional. She lived and worked in London, UK for the past decade and just moved back to her home town Salburg in Austria. She is an avid skier but also loves to spend quality time on the beach during the warm weather. Travelling is one of her favourite hobbies as well as spending time with family and friends.'
+
+victoria_fitoussi:
+  name: Victoria Fitoussi
+  email: victoria.fitoussi@sourcegraph.com
+  github: vfitoussi
+  pronouns: she/her
+  role: Senior Content Marketing Manager
+  reports_to: VP_marketing
```

Comment:
@vfitoussi the yaml here didn't pass build, I _think_ because `VP_marketing` should be `vp_marketing`. Since you'll shortly revert to reporting to Erica, I would go ahead and set this to `director_content_strategy` so that the build system correctly places you in our team on the org chart.